content/en/blog/_posts/2022-11-28-kubelet-credential-providers/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: blog
+title: 'Kubernetes v1.26: GA Support for Kubelet Registry Credential Providers'
+date: 2022-11-28
+slug: kubelet-credential-providers
+---
+
+**Authors:** Andrew Sy Kim (Google), Dixita Narang (Google)
+
+Kubernetes v1.26 introduces GA support for Kubelet Credential Providers,
+offering an extensible plugin to dynamically fetch credentials for any container registry service.
+
+## Background
+
+Kubernetes supports the ability to dynamically fetch credentials for a container registry service.
+However, prior to Kubernetes v1.20, this capability was built into the Kubelet and only available for
+Amazon Elastic Container Registry, Azure Container Registry, and Google Cloud Container Registry.
+
+{{< figure src="kubelet-credential-providers-in-tree.png" alt="Figure 1: Kubelet Built-in Credential Provider Support for Amazon Elastic Container Registry, Azure Container Registry, and Google Cloud Container Registry." >}}
+
+Kubernetes v1.20 introduced alpha support for Kubelet Credential Providers, providing a mechanism for
+the Kubelet to dynamically authenticate and pull images for any container registry service.
+In Kubernetes v1.26, this feature is now GA.
+
+{{< figure src="kubelet-credential-providers-plugin.png" alt="Figure 2: Kubelet Credential Provider Overview" >}}
```

Comment:
@sftim is there a way to add description text for figures?

Reply:
See https://gohugo.io/content-management/shortcodes/#figure

There are a few options.